wgsl/index.bs

```diff
@@ -8627,6 +8632,36 @@ where compatibility is defined by the following table.
 See the [$validating GPUProgrammableStage|WebGPU API$]
 specification for interface validation requirements.
 
+### Buffer Binding Determines Runtime-Sized Array Element Count ### {#buffer-binding-determines-runtime-sized-array-element-count}
+
+When a [=storage buffer=] variable contains a [=runtime-sized=] array, then the number of elements in that array
+is determined from the size of the corresponding {{GPUBufferBinding}}:
+
+<blockquote algorithm="element count of runtime-sized array">
+* Let |T| be the [=store type=] for a [=storage buffer=] variable,
+    where |T| is a [=runtime-sized=] array type or contains a runtime-sized array type.
+* Let |EBS| be the
+    [=effective buffer binding size=]
+    for the {{GPUBufferBinding}} bound to the pipeline binding address corresponding to the storage buffer variable.
+* Then <dfn noexport>NRuntime</sub></dfn>, i.e.
+    the number of elements in the runtime-sized array,
+    is the largest integer such that [=SizeOf=](|T|) &le; |EBS|.
```

Comment:
If the runtime-sized array is inside a structure, you need to take the byte-offset of that struct member into account.

Reply:
This does.  |T| is not the runtime-array, it's the store type of the whole variable.
The sizeof equation here takes the whole recursive size computation into account.

Reply:
Ah yes, sorry.